consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftController.java

```diff
@@ -248,5 +254,9 @@ private void addMessageToFutureMessageBuffer(final long chainHeight, final Messa
       futureMessages.put(chainHeight, Lists.newArrayList());
     }
     futureMessages.get(chainHeight).add(rawMsg);
+
+    // Notify the synchroniser the transmitting peer must have the parent block to the received
+    // message's target height.
+    sychronizerUpdater.updatePeerChainState(chainHeight - 1L, rawMsg.getConnection());
```

Comment:
Do we also need to call updatePeerChainState when we receive a new block event? I would hope that we don't need but not sure.

Reply:
No we don't - that's already been handled by the BlockPropogation mechanism.

Reply:
fair enough